### PR TITLE
Add commas after skipped arms when needed

### LINKED ISSUE #737: Comma stripped from a #[rustfmt_skip]'d match arm
```diff
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,7 +1,7 @@
 fn foo() {
     let _ = match Some(1) {
         #[rustfmt_skip]
-        Some(n) => n,
+        Some(n) => n
         None => 1,
     };
 }
```

### PR DESCRIPTION
**Pull request by sanxiyn**
Fix #737.

### CODE REVIEW

File: tests/source/match.rs [hunk 1]
```diff
@@ -100,6 +100,14 @@ fn matches() {
     }
 }
 
+fn match_skip() {
+    let _ = match Some(1) {
+        #[rustfmt_skip]
+        Some(n) => n,
```
Comment (marcusklaas) [inline tests/source/match.rs]: Could you change both patterns to something non-canonical to check that rustfmt does not change the `Some` line, but does reformat the `None` line?

**Comment by marcusklaas** [comment]
Looks good to me. r+ with updated test.

**Comment by sanxiyn** [comment]
Test updated.

**Comment by marcusklaas** [comment]
Thanks!

**Pull request merged**